pkg/runner/cluster_k8s.go

```diff
@@ -293,6 +295,131 @@ func (*ClusterK8sRunner) ID() string {
 	return "cluster:k8s"
 }
 
+func (c *ClusterK8sRunner) healthcheckK8s() (k8sCheck api.HealthcheckItem) {
+	k8sCheck = api.HealthcheckItem{Name: "k8s", Status: api.HealthcheckStatusOK, Message: "k8s cluster is running"}
+	err := exec.Command("sh", "-c", "kops validate cluster").Run()
+	if err != nil {
+		k8sCheck = api.HealthcheckItem{Name: "k8s", Status: api.HealthcheckStatusFailed, Message: fmt.Sprintf("k8s cluster validation failed: %s", err)}
+		return
+	}
+	return
+}
+
+func (c *ClusterK8sRunner) healthcheckEFS() (efsCheck api.HealthcheckItem) {
+	efsCheck = api.HealthcheckItem{Name: "efs", Status: api.HealthcheckStatusFailed, Message: "efs provisioner is not running"}
+
+	client := c.pool.Acquire()
+	defer c.pool.Release(client)
+
+	res, err := client.CoreV1().Pods(c.config.Namespace).List(metav1.ListOptions{
+		LabelSelector: "app=efs-provisioner",
+	})
+	if err != nil {
+		return
+	}
+	if len(res.Items) != 1 {
+		return
```

Comment:
Should override `efsCheck.Message` with a string like: "expected 1 EFS provisioner pod; found: $len(res.Items)"